tests/rptest/tests/enterprise_features_license_test.py

```diff
@@ -151,6 +157,8 @@ def try_enable_feature(self, feature):
             self.redpanda.set_cluster_config(
                 {'sasl_mechanisms': ['SCRAM', 'OAUTHBEARER']})
         elif feature == Feature.schema_id_validation:
+            # Note: the default OIDC server is flaky in CI, so use `OIDC_UPDATE_FAILURE_LOGS`
+            # to ignore the related error logs in the background updater.
```

Comment:
nitpick: looks like this comment should go on the previous "switch" case

Reply:
Yeah good spot. Fixed